### PR TITLE
Background, development, introduction, mysqloperator chapters refactor

### PR DESCRIPTION
**Pull request by jsarzy**

### CODE REVIEW

File: src/development.tex [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-development stack.
+the development stack.
 
 \section{Version Control System}
-Our VCS of choice is \textbf{Git}, and our repositories are hosted on 
-\textbf{GitHub}. Being the most popular service, GitHub is well integrated 
-with many developer tools such as Golang’s \texttt{go get} and various 
-Continuous Integration systems. On top of that, we use common GitHub features 
-for managing current tasks and project development.
+Our V.C.S of choice is \textbf{Git} with the \textbf{GitHub}~\footnote{https://github.com} hosting.
```
Comment (aswietlik) [inline src/development.tex]: I think we wanted to put `version control system` here instead of `V.C.S.`

File: src/introduction.tex [hunk 2]
```diff
@@ -7,59 +7,58 @@ \chapter{Introduction}
...
+abstraction system provided by Kubernetes in such a way will remove
 much of the complexity traditionally caused by the database layer of
 an~application.
 
 There are existing unofficial tools with similar functionalities for various
-databases and systems. \textit{Vitess}~\cite{vitess} is an~open source
-clustering system for \textit{MySQL} databases that can interface with
-\textit{Kubernetes}. We are aiming for a simpler solution that utilizes
-\textit{Kubernetes'} native method of extending its API ---
+databases and systems. Vitess~\cite{vitess} is an~open source
```
Comment (aswietlik) [inline src/introduction.tex:39]: Don't we want to add \textbf{vitess} here? It's a definition, right?

File: src/introduction.tex [hunk 3]
```diff
@@ -7,59 +7,58 @@ \chapter{Introduction}
...
 
 We were working together on all parts of the implementation, however there are 
 some areas which certain team members were responsible for. Marcin Chrzanowski 
-was involved in Stateful Set and MySQL Cluster creation and deletion. Jakub 
-Sarzyński’s main task was to provide a Command Line Interface for our tools. 
-Agnieszka Świetlik was responsible for backups design and implementation. Mikołaj 
-Walczak was the core of our team, he was in charge of Continuous Integration, 
+was involved in MySQL' Cluster creation and deletion. Jakub 
+Sarzyński’s main task was to provide a command line interface. 
+Agnieszka Świetlik was responsible for the implementation of backups, design. Mikołaj
```
Comment (aswietlik) [inline src/introduction.tex]: `the design and implementation of backups`?

File: src/background.tex [hunk 4]
```diff
@@ -66,7 +66,7 @@ \section{Expanding the Kubernetes API}
 
 The first way of expanding the Kubernetes API, introduced in Kubernetes 1.7, is using the
 aggregation layer. Thanks to this tool, one can perform API aggregation at runtime. The aggregation
-layer enables the creation of custom resources by deploying its own API server.
+layer enables the creation of Custom Resources by deploying its own API server.
```
Comment (aswietlik) [inline src/background.tex]: Either `CustomResources` or `custom resources`, everywhere.

File: src/introduction.tex [hunk 5]
```diff
@@ -7,59 +7,58 @@ \chapter{Introduction}
...
+Crunchy Data PostgreSQL Operator~\cite{psql-op}, is closer in scope to what we want to
+accomplish, providing a CRD for Postgres databases.
 
 This dissertation is divided into eight chapters. We start with the necessary 
 preliminary knowledge about Kubernetes and Custom Resources. Next, we cover 
 our workflow and development stack. The main chapter describes the design and 
-implementation details of the MySQL Operator functionalities. The following 
-chapter consists of essential information of the command line interface we 
+implementation details of the MySQL Operator's functionalities. The following 
+chapter cover the command line interface we
```
Comment (aswietlik) [inline src/introduction.tex]: `covers`

File: src/mysqloperator.tex [hunk 6]
```diff
@@ -20,31 +20,31 @@ \section{High-level overview}
 \subsection{Functionalities}
 The MySQL Operator allows the user to create, delete, update, backup and restore MySQL clusters.
 
-When \textbf{creating} the cluster, the operator deploys a new instance of a replicated MySQL
```
Comment (aswietlik) [inline src/mysqloperator.tex:23]: Maybe `textit` for the previously bolded parts? We're not using this decoration anywhere.

File: src/introduction.tex (hunk 2 above)
Comment (m-chrzan) [reply src/introduction.tex:39]: We never mention it again in the thesis though, so it is a definition by definition, but not a definition we really care about.

File: src/background.tex (hunk 4 above)
Comment (m-chrzan) [reply src/background.tex]: "Custom resources", definitely not "CustomResources" - there's no k8s object with that name.

File: src/background.tex [hunk 7]
```diff
@@ -39,7 +39,7 @@ \section{Kubernetes Architecture}
 \end{figure}
 
 To get a better understanding of Kubernetes structure, let’s avail ourselves of the data presented
-in the Figure \ref{fig:arch}. Each worker node is equipped with a \textit{kubelet} program
+in the Figure \ref{fig:arch}. Each worker node is equipped with a \textbf{kubelet} program
```
Comment (m-chrzan) [inline src/background.tex]: "... equipped with a program called kubelet, responsible for ..."

File: src/background.tex [hunk 8]
```diff
@@ -56,7 +56,7 @@ \section{Kubernetes Architecture}
 
 \section{Expanding the Kubernetes API}
 Kubernetes was designed to be easily extensible and configurable~\cite{extending-kubeapi}. There are
-many ways of customizing the API and adding new functionalities, so called Custom Resources.
+many ways of customizing the API and adding new functionalities, so called \textbf{Custom Resources}.
```
Comment (m-chrzan) [inline src/background.tex]: "custom resources"

File: src/development.tex [hunk 9]
```diff
@@ -1,14 +1,14 @@
...
+Being the most popular service, GitHub is well integrated  with many developer
+tools such as Golang’s \texttt{go get} and various Continuous Integration
```
Comment (m-chrzan) [inline src/development.tex]: "continuous integration"

File: src/introduction.tex [hunk 10]
```diff
@@ -7,59 +7,58 @@ \chapter{Introduction}
...
 design decisions, obstacles encountered, and future considerations for further 
 work on the project.
 
 We were working together on all parts of the implementation, however there are 
 some areas which certain team members were responsible for. Marcin Chrzanowski 
-was involved in Stateful Set and MySQL Cluster creation and deletion. Jakub 
-Sarzyński’s main task was to provide a Command Line Interface for our tools. 
-Agnieszka Świetlik was responsible for backups design and implementation. Mikołaj 
-Walczak was the core of our team, he was in charge of Continuous Integration, 
+was involved in MySQL' Cluster creation and deletion. Jakub
```
Comment (m-chrzan) [inline src/introduction.tex]: "MySQLCluster"

File: src/mysqloperator.tex [hunk 11]
```diff
@@ -66,16 +66,16 @@ \subsection{Design Patterns}
...
 example of how an operator will respond to a user’s action. Let’s say the user wants to modify a
-\texttt{MySQLCluster} instance. From the user’s point of view, this can be done by issuing a
-\texttt{kubectl patch} command. From the operator’s point of view, an \textit{Update} event is
+MySQLCluster instance. From the user’s point of view, this can be done by issuing a
+\texttt{kubectl patch} command. From the operator’s point of view, an \textbf{Update} event is
 received and the corresponding operator handler is called by the controller.
 
 We use the controller and operator patterns to manage resources related to both 
-\texttt{MySQL servers} and \texttt{backup jobs}. A high level overview of the project design is
+\textbf{MySQL servers} and \textbf{backup jobs}. A high level overview of the project design is
```
Comment (m-chrzan) [inline src/mysqloperator.tex]: Do we need bold on either of these?

**Review by m-chrzan — CHANGES_REQUESTED** [review]

**Review by m-chrzan — APPROVED** [review]

**Pull request merged**